wallet-new/src/Cardano/Wallet/Kernel/DB/AcidState.hs

```diff
@@ -217,8 +222,7 @@ createPrefiltered initUtxoAndAddrs applyP accs = do
                 , _checkpointUtxoBalance = InDb $ Spec.balance utxo'
                 , _checkpointExpected    = InDb Map.empty
                 , _checkpointPending     = Pending . InDb $ Map.empty
-                -- TODO(@uroboros/ryan) proper BlockMeta initialisation (as part of CBR-239: Support history tracking and queries)
-                , _checkpointBlockMeta   = BlockMeta . InDb $ Map.empty
+                , _checkpointBlockMeta   = mempty::BlockMeta
```

Comment:
Weird, why did you need the explicit type annotation here? Wasn't `mempty` enough?

Reply:
`mempty` was enough, yes, I was just being explicit (I'm removing it though)

Reply:
That's a laudable goal -- I usually resort to an explicit (yet banal) "smart constructor" like `newBlockMeta`, but here I guess the fact `mempty` is contextualised by the `_checkpointBlockMeta` field makes things explicit enough 😉 

I reckon it's just a matter of preferences, though!